deepchecks/tabular/context.py

```diff
@@ -144,6 +145,9 @@ class Context:
         The name of the model
     features_importance: pd.Series , default: None
         pass manual features importance
+        (parameter is deprecated, use 'feature_importance' instead)
+    feature_importance: pd.Series , default: None
+        pass manual features importance
```

Comment:
I think you can use the .. deprecated directive

Reply:
fixed